scss/individual/patterns_lists.scss

```diff
@@ -0,0 +1,9 @@
+@import '../base';
+@include vf-base;
+
+// vf-p-stepped-list-detailed requires some grid mixins and placeholders
+@import '../patterns_grid';
```

Comment:
I've been wondering the same thing for a while, and these are the arguments for / against including it in base that I've managed to come up with:

For:
Not sure where it should fit architecturally, but from the perspective of usage, does every site using vanilla rely on it? AFAIK the answer is yes. If so, then putting it in base wouldn't lead to an increased file size, and would avoid further complication like adding a separate category for it.

Against:
However, if we were to add an ie 11 fallback in the future (e.g. some limited flex grid), would the associated file size increase be too much for sites that have no ie11 users?

Reply:
I would suggest that placeholders be separated into a different file and included via base. As they do not add any more file size to the output and should be namespaces to remove conflicts. 

Please, do not make the grid a base element as components are agnostic to the grid but taking up the width of there container. It might be that someone wants to use there own grid and we should allow for that.